OSD.adoc

```diff
@@ -13,6 +13,12 @@ Welcome, Open Source Day participants! We are glad you have joined us!
 
 To get started, follow the steps below:
 
+. *Create a Jira Account.* Visit https://issues.redhat.com/ and click on *Log In* at the top right corner of the page.
+* At the bottom of the *Log in to your Red Hat account* section, click on *Register for a Red Hat account*, which will lead you to a page to create an account. 
+* Fill in all the fields accordingly and click on *Create my account*. 
+* You will be asked to verify your email using a link sent to the email you used to register.
+* Once you have successfully created and verified your account, you can go back to https://issues.redhat.com/ to log into your Jira account.
```

Comment:
One last thing - maybe it would be good to mention that this account will be used to assign issues to yourself in step 4. 

Or maybe in step 4 mention that they need to be logged into to Jira using the account created in step 1.

Whichever makes the most sense is fine.

WDYT?

Reply:
I have mentioned step 4 in step 1 for both
Thanks!